dbeam-core/src/main/java/com/spotify/dbeam/args/QueryBuilderArgs.java

```diff
@@ -99,8 +105,14 @@ public static QueryBuilderArgs create(String tableName) {
                                    partitionColumn, datePartition, partitionColumn, nextPartition);
             })
     ).orElse("");
+    String fieldsList;
+    if (!fields().isPresent()) {
+      fieldsList = "*";
+    } else {
+      fieldsList = StringUtils.join(fields().get(), ',');
+    }
     return Lists.newArrayList(
-        String.format("SELECT * FROM %s%s%s", this.tableName(), where, limit));
+        String.format("SELECT %s FROM %s%s%s", fieldsList, this.tableName(), where, limit));
```

Comment:
Where can these fields come from? This statement is vulnerable to SQL injections, and can be solved in a safe manner.

Reply:
@VibyJocke Really good comment. The job is a command line tool with no exposure towards endusers. The field names comes via the command line or to be more precise the airflow job scheduler in our case. Access to databases is restricted to read only. 
That said it is a good idea to do an extra validation of the command line input to ensure it only includes [a-zA-Z0-9_-]

Reply:
Ok, since it's a command-line tool I'll leave it up to you if you want to fix it 😅 The only risk is that someone might expose/rework it in some way in the future without proof-reading all code.